pkg/build/controller/build/build_controller.go

```diff
@@ -2,6 +2,7 @@ package build
 
 import (
 	"fmt"
+	"github.com/openshift/api/config/v1"
```

Comment:
nit - openshift imports should be grouped together and in alpha order.

Reply:
fixed in a recent push, didn't realize my ide imported it and put it there.